xml/ha_storage_protection.xml

```diff
@@ -698,6 +698,66 @@ Timeout (msgwait)  : 10
    </procedure>
   </sect1>
 
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting up Diskless SBD</title>
+   <para>
+    SBD can be operated in a diskless mode. As diskless SBD is not able
```

Comment:
The sentence "As diskless SBD is not able to ..." is hard to understand. In addition, I would make it more prominent that it must not be used for two-node clusters. Maybe put it like this:
```
<para>SBD can be operated in a diskless mode.</para>
    <important>
     <title>Number of Cluster Nodes</title>
       <para>
         Do <emphasis>not</emphasis> use diskless SBD as fencing mechanism for two-node clusters. 
         Use it only in clusters with three or more nodes. SBD in diskless mode cannot handle split brain
         scenarios for two-node clusters.  
      </para>
   </important>```